.authors.yml

```
@@ -2531,3 +2531,13 @@
   github: v2thegreat
   num_commits: 1
   first_commit: 2025-02-11 10:38:55
+- name: Faith
+  email: 85044320+faithrider@users.noreply.github.com
+  num_commits: 2
+  first_commit: 2025-05-13 10:20:45
+  github: faithrider
+- name: Peyton Murray
+  email: peynmurray@gmail.com
+  num_commits: 1
+  first_commit: 202$-05-17 13:40:33
```

What is this dollar sign? Should it be a 5?

```suggestion
  first_commit: 2025-05-17 13:40:33
```

---

Yeah. Not sure how that got there thougeh.